src/main/java/com/example/sharemind/customer/application/CustomerServiceImpl.java

```diff
@@ -0,0 +1,25 @@
+package com.example.sharemind.customer.application;
+
+import com.example.sharemind.customer.domain.Customer;
+import com.example.sharemind.customer.exception.CustomerErrorCode;
+import com.example.sharemind.customer.exception.CustomerException;
+import com.example.sharemind.customer.repository.CustomerRepository;
+import lombok.RequiredArgsConstructor;
+import org.springframework.stereotype.Service;
+import org.springframework.transaction.annotation.Transactional;
+
+@Service
+@Transactional(readOnly = true)
+@RequiredArgsConstructor
+public class CustomerServiceImpl implements CustomerService {
+
+    private final CustomerRepository customerRepository;
+
+    @Override
+    public String getCustomerNickname(Long customerId) {
```

Comment:
여기도 마찬가지로 Customer를 넘겨주면 어떨까요!?
```suggestion
    public Customer getCustomerByCustomerId(Long customerId) {
```